syne_tune/experiments/launchers/hpo_main_sagemaker.py

```diff
@@ -146,7 +147,6 @@ def start_benchmark_sagemaker_backend(
     )
     del sm_args["checkpoint_s3_uri"]
     sm_args["sagemaker_session"] = default_sagemaker_session()
-    sm_args["dependencies"] = benchmarking.__path__
```

Comment:
This is replaced by `source_dependencies`, as the user may not even have `benchmarking` (if not installed frmo source).